molecularnodes/io/universe/universe.py

```diff
@@ -0,0 +1,517 @@
+from typing import Dict, List
+
+import bpy
+import MDAnalysis as mda
+import numpy as np
+import numpy.typing as npt
+from bpy.app.handlers import persistent
+
+from ... import data
+from ...types import MNDataObject, ObjectMissingError
+from ...blender import coll, nodes, obj
+from ...utils import lerp, correct_periodic_positions
+from .selections import Selection, TrajectorySelectionItem
+
+
+class MNUniverse(MNDataObject):
+    def __init__(self, universe: mda.Universe, world_scale=0.01):
+        super().__init__()
+        self.universe: mda.Universe = universe
+        self.selections: Dict[str, Selection] = {}
+        self.world_scale = world_scale
```

Comment:
This could be set in the `MNDataObject` - thinking about it and the potential to change it after the fact - this should really be a `bpy.context.scene.MN_world_scale` property, so that when it changes it can be access from anywhere. This way it could be changed inside of node trees when the property is updated also. Unsure if we should actually want to support this though, as it has the potential to get messy. It is easy enough for users to just scale the actual object inside of the Blender scene instead.

I think in the meantime this can be moved to the `MNDataObject` though you are right. There are still lots of little bits of cleanup / optimisation like that to be done around the place.

Reply:
> It is easy enough for users to just scale the actual object inside of the Blender scene instead.

Yes, that’s what I thought—it seems to be a duplicate of the scale in Blender the way it’s currently set. I think setting a `bpy.context.scene.MN_world_scale` property, as you mentioned, makes more sense, and maybe set it as a property of MNSession in the meantime.